diagnostic.md

```diff
@@ -19,8 +25,11 @@ c = c + d
 ```
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d` and explain what they do.
+<!-- solution below -->
+Value of C is 7
 
-
+The last line is evaluating from right to left so after it adds the numbers it
+reassings the value to c
```

Comment:
The two operators are `=` for assigning value and `+` for adding two values together.